frontend/ui/elements/menu_keyboard_layout.lua

```diff
@@ -224,7 +227,27 @@ local sub_item_table = {
         end,
     },
 }
+-- until SDL shenanigans are fixed (no input from physical keys when vk not visible), we won't make it available there
+if (Device:hasKeyboard() and not Device.isSDL()) or Device:hasScreenKB() then
```

Comment:
```suggestion
-- until SDL shenanigans are fixed (no input from physical keys when vk not visible), we won't make it available there
if (Device:hasKeyboard() and Device.isEmulator()) or Device:hasScreenKB() then
```

Reply:
You sure about that one? HasKB and isEmulator?

Reply:
The emulator is for development. Hiding things from development is an excellent way to ensure it's never touched again. ;-)

Reply:
Still you’d be hiding it on kindle keyboards then, which is largely against the point behind of this PR.

Reply:
Could also temporarily be `(HasKB and isEmulator) or hasScreenKB or hasSymKey` with a note to remove the hasSymkey once the SDL is fixed.

Reply:
Or it could be greyed out, that way it would still be a pebble in the shoe

Reply:
Let's just write it properly.
```suggestion
if Device:hasKeyboard() or Device:hasScreenKB() then
```